util/automation.py

```diff
@@ -115,14 +135,20 @@ def get_new_commits(sha1_from):
         return None, r.status_code
     payload = json.loads(r.text)
 
+    this_merge_date = git_date(payload['base_commit']['commit']['committer']['date'])
+
     if payload['status'] == 'identical':
         # When there is no change, just nicely output no new commits
         return [], r.status_code
     elif payload['status'] == 'ahead':
         # Let us just keep the merges commits, we can identify them because they
         # are the commits with two parents
-        merges = filter(lambda x: len(x['parents']) >= 2, payload['commits'])
-        return map(lambda x: x['sha'], merges), r.status_code
+        all_merges = filter(lambda x: len(x['parents']) >= 2, payload['commits'])
+        # Keep only merges made from Github UI
+        gh_merges = filter(lambda x: is_webflow(x), all_merges)
```

Comment:
Because that is the simplest way I found to identify a merge from a PR rather than a merge that you could have done locally :)